proposal/proposal.tex

```diff
@@ -7,7 +7,7 @@
 source code meta-programming (Camlp4/5). Since then, meta-programming is mostly
 done on the parsetree and has incrementally evolved into a solid ecosystem,
 enriching the OCaml developer experience. Two highlights of that evolvement:
-introducing parsetree migrations to gain individual cross-compiler compatibility
+introducing parsetree migrations to gain individual cross-version compatibility
```

Comment:
What would you think about "cross-compiler-version" here and everywhere else? I'm not sure whether it's clear to everyone that we're talking about compatibility regarding the compiler when only saying "cross-version". Do you think it would be clear to everyone?
```suggestion
introducing parsetree migrations to gain individual cross-compiler-version compatibility
```